src/main/java/hello/Task.java

```diff
@@ -82,46 +65,55 @@ public String toString() {
         return "Task{" +
                 "id=" + id +
                 ", code='" + code + '\'' +
-                ", status=" + status +
+                ", Status=" + status +
                 ", consoleOutput='" + consoleOutput + '\'' +
+                ", exception='" + exception + '\'' +
                 '}';
-
-    public boolean equals(Object o) {
-        if (this == o) return true;
-        if (o == null || getClass() != o.getClass()) return false;
-
-        Task task = (Task) o;
-
-        if (isConsoleOutputOK != task.isConsoleOutputOK) return false;
-        if (id != null ? !id.equals(task.id) : task.id != null) return false;
-        if (code != null ? !code.equals(task.code) : task.code != null) return false;
-        if (scriptStatus != task.scriptStatus) return false;
-        if (consoleOutput != null ? !consoleOutput.equals(task.consoleOutput) : task.consoleOutput != null)
-            return false;
-        return exception != null ? exception.equals(task.exception) : task.exception == null;
-
-    }
-
-    @Override
-    public int hashCode() {
-        int result = id != null ? id.hashCode() : 0;
-        result = 31 * result + (code != null ? code.hashCode() : 0);
-        result = 31 * result + (isConsoleOutputOK ? 1 : 0);
-        result = 31 * result + (scriptStatus != null ? scriptStatus.hashCode() : 0);
-        result = 31 * result + (consoleOutput != null ? consoleOutput.hashCode() : 0);
-        result = 31 * result + (exception != null ? exception.hashCode() : 0);
-        return result;
     }
 
     @Override
-    public String toString() {
-        return "Task{" +
-                "id=" + id +
-                ", code='" + code + '\'' +
-                ", isConsoleOutputOK=" + isConsoleOutputOK +
-                ", scriptStatus=" + scriptStatus +
-                ", consoleOutput='" + consoleOutput + '\'' +
-                ", exception='" + exception + '\'' +
-                '}';
+    public synchronized Status call() throws Exception {
+        try {
+        	status(Status.RUNNING);
+            log.debug(getId() + " started in thread " + Thread.currentThread().getName());
+            final ScriptContext ctx = new SimpleScriptContext();
+            ctx.setWriter(consoleOutput);
+            ctx.setErrorWriter(consoleOutput);
+            ctx.setAttribute("id", this.id.toString(), ScriptContext.ENGINE_SCOPE);
```

Comment:
we can create context and set variables, which can be used from script, try printing id from script